CMakeLists.txt

```diff
@@ -1037,6 +1037,7 @@ check_include_file_concat("signal.h"         HAVE_SIGNAL_H)
 check_include_file_concat("stdatomic.h"      HAVE_STDATOMIC_H)
 check_include_file_concat("stdbool.h"        HAVE_STDBOOL_H)
 check_include_file_concat("stdint.h"         HAVE_STDINT_H)
+check_include_file_concat("stdio.h"          HAVE_STDIO_H)
```

Comment:
Why was this check added?

`HAVE_STDIO_H` is not used in the source. `stdio.h` is a standard C89 header and we use it everywhere unconditionally.

Reply:
yes, it is the Fseek detection that needs it.

Reply:
Is that due to some funky cmakeism? We don't have to check for stdio.h, but maybe it needs to be set somehow so that it gets used for the fseeko check?